index.bs

```diff
@@ -1341,9 +1361,11 @@ To <dfn lt="updating the event map">update the event map</dfn>, given
     filter only those that ought to be returned to the local end.
 </div>
 
-### Status ### {#command-session-status}
+### Commands ### {#module-session-commands}
 
-The <dfn for=commands>status</dfn> command returns information about
+#### Status #### {#command-session-status}
```

Comment:
```suggestion
#### The session.status Command #### {#command-session-status}
```

As per above. Or we just uppercase "session" above again :)

Ditto in the follow, I won't make suggestions throughout.